app.py

```diff
@@ -1,20 +1,135 @@
 from flask import Flask, render_template, request, redirect, url_for
+import config as cf
+from model import Item
+
 import session_items as session
+import requests
+import json
+
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
+#load_dotenv()
 
 @app.route('/')
 def index():
-    items = session.get_items()
-    return render_template('index.html', items=items)
-    #return 'Hello World!'
+    url = "https://api.trello.com/1/boards/{id}/cards"
+    query = cf.get_trello_query()
+    
+    response = requests.request(
+        "GET",
+        url,
+        params=query
+        )
+    #print(response.text)
+    #items = session.get_items()
+    #return render_template('index.html', items=items)
+    cards = json.loads(response.text)
+    items_list = list()
+    for card in cards:
+        item = Item(card.id, card.status, card.title)
```

Comment:
Even if you fix the issue, with the URL, `card` here is a dictionary. You need to access its values using e.g. `card['id']`. It does not have "status" or "title" entries.